src/Dialect/ONNX/CMakeLists.txt

```diff
@@ -14,12 +14,19 @@ add_custom_target(OMONNXOpsInc
         DEPENDS OMONNXOpsIncGen
                 OMResultTypeInferenceOpInterfaceIncGen
                 OMShapeInferenceOpInterfaceIncGen
-                OMHasOnnxSubgraphOpInterfaceIncGen)
+                OMHasOnnxSubgraphOpInterfaceIncGen
+                OMONNXRewriteIncGen
+                OMONNXCombineIncGen)
+
+add_onnx_mlir_rewriter(Rewrite)
```

Comment:
Move these before adding the custom target since the custom target depends on the two targets created by the add calls

Reply:
done